next/wunder-component-library/skip-to-content-link.tsx

```diff
@@ -0,0 +1,19 @@
+import Link from "next/link";
+import React from "react";
+
+interface ContentLinkProps {
+  href: string;
+  children: React.ReactNode;
+}
+
+export function SkipToContentLink({ href, children }: ContentLinkProps) {
+  return (
+    <Link
+      href={href}
+      className="sr-only font-inter tracking-wide text-primary-600 underline-offset-4 focus-visible:not-sr-only focus-visible:underline focus-visible:ring focus-visible:ring-primary-600"
```

Comment:
This works nicely but as discussed, we can probably move most of these focus-visible styles as they're handled globally by _focus-visilble.css. (It might be that `focus-visible:not-sr-only` is the only required one?)